### PR TITLE
Gives spoons `TOOL_MINING`, adds spoons to autolathe and prison vending machines

### PR DESCRIPTION
**Pull request by TemporalOroboros**
## About The Pull Request
- Implements `/obj/item/kitchen/spoon`
- Gives spoons `TOOL_MINING`

## Why It's Good For The Game
- This is one of the most basic prison memes, why do we not have this

## Changelog
:cl:
add: You can dig through rock and dirt with a spoon, elbow grease, and a <i>lot</i> of time
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
now add a super long tunnel with just rocks from brig to armory

**Comment by D4C-420** [comment]
Put these in everyone's internals boxes on icebox. Would make it suck less to fall down a z level or two on ice box.

**Comment by Qustinnus** [comment]
lol just grab a spoon

**Comment by D4C-420** [comment]
if your going out of your way to grab a spoon why not grab a plasma cutter from cargo? But yeah fair enough.

**Comment by justinbooj9** [comment]
contractible mining spoon now available in your emergency internals box

**Comment by ArcaneMusic** [comment]
Plastic Spoons are printable at all departmental lathes, as well as autolathes.

**Comment by D4C-420** [comment]
you could say the same thing about internals. Emergency oxy tanks, and face masks are abundant in the station along with gas mask and full oxy tanks but we still give everyone internals why? Because dying to a single hull breach or n2o canister opening is not fun and having large parts of the station rendered uninhabitable because you cant completely remove the gas is also not fun. Getting rid of everyone's internals boxes would just make people steal all the other ones see #54689 for why people argued it was not merged. Giving everyone a plastic spoon in there internals boxes would fix iceboxes largest problem of people just being stuck in walls and pretty much round removed because a single decent size bomb went off rending parts of a hall way a death trap. Sure they would still have to make their way back to the station but now they at least have a chance instead of being stuck in a wall unable to do anything and just ssding because they did not go-ahead and bitch to chem to make plastic just so they can have a small, slow mining tool in case they need it.

**Review by Mothblocks — DISMISSED** [review]
emphasis on your environment babyyyyy

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -276,15 +276,33 @@
...
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 50, ACID = 30)
+	custom_materials = list(/datum/material/iron=120)
+	custom_price = PAYCHECK_PRISONER * 5
+	tool_behaviour = TOOL_MINING
+	toolspeed = 25 // Literally 25 times worse than the base pickaxe
+
 /obj/item/kitchen/spoon/plastic
 	name = "plastic spoon"
 	desc = "Just be careful your food doesn't melt the spoon first."
 	icon_state = "plastic_spoon"
```
Comment (Jared-Fogle) [inline code/game/objects/items/kitchen.dm:299]: remove some of these since they're duped now

File: code/game/objects/items/kitchen.dm [hunk 2]
```diff
@@ -276,15 +276,33 @@
...
+	w_class = WEIGHT_CLASS_TINY
+	flags_1 = CONDUCT_1
+	force = 2
+	throw_speed = 3
+	throw_range = 5
+	attack_verb_simple = list("whack", "spoon", "tap")
+	attack_verb_continuous = list("whacks", "spoons", "taps")
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 50, ACID = 30)
+	custom_materials = list(/datum/material/iron=120)
+	custom_price = PAYCHECK_PRISONER * 5
```
Comment (Jared-Fogle) [inline code/game/objects/items/kitchen.dm:292]: this has a price, but is it in the vending machines?

**Comment by ilikesaying** [comment]
**make digging with your fingers a last resort, taking more time, elbow grease and damage to arms**

File: code/game/objects/items/kitchen.dm (hunk 1 above)
Comment (TemporalOroboros) [reply code/game/objects/items/kitchen.dm]: Done

File: code/game/objects/items/kitchen.dm (hunk 2 above)
Comment (TemporalOroboros) [reply code/game/objects/items/kitchen.dm:292]: Now it is

**Pull request merged**